ingesters/ingesters.md

```diff
@@ -12,6 +12,29 @@ The Gravwell GUI has an Ingesters page (under the System menu category) which ca
 
 Attention: The [replication system](#!configuration/replication.md) does not replicate entries larger than 999MB. Larger entries can still be ingested and searched as usual, but they are omitted from replication. This is not a concern for 99.9% of use cases, as all the ingesters detailed in this page tend to create entries no larger than a few kilobytes.
 
+## Ingesters
+
+| Ingester | Description |
+|----------|-------------|
+| [Amazon SQS](#!ingesters/sqs.md) | Subscribe and ingest from Amazon SQS queues. |
+| [Disk Monitor](#!ingesters/disk.md) | Periodically sample disk activity. |
+| [File Follower](#!ingesters/file_follow.md) | Watch and ingest files on disk, such as logs. |
+| [GCP PubSub](#!ingesters/pubsub.md) | Fetch and ingest entries from Google Compute Platform PubSub Streams. |
+| [HTTP](#!ingesters/http.md) | Create HTTP listeners on multiple URL paths. |
+| [IPMI](#!ingesters/ipmi.md) | Periodically collect SDR and SEL records from IPMI devices. |
+| [Kafka](#!ingesters/kafka.md) | Create a Kafka Consumer that ingests into Gravwell. Can be paried with the Gravwell Kafka Federator. |
+| [Kinesis](#!ingesters/kinesis.md) | Ingest from Amazon's [Kinesis Data Streams](https://aws.amazon.com/kinesis/data-streams/) service. |
+| [Mass File](#!ingesters/massfile.md) | Ingest large numbers of static files. |
+| [Microsoft Graph API](#!ingesters/msg.md) | Ingest from Microsoft's Graph API. |
+| [Netflow](#!ingesters/netflow.md) | Collect Netflow and IPFIX records. |
+| [Network Capture](#!ingesters/pcap.md) | Ingest PCAP on the wire. |
+| [Office 365](#!ingesters/o365.md) | Ingest Microsoft o365 Logs. |
+| [Packetfleet](#!ingesters/packetfleet.md) | Issue queries and ingest data from Google Stenographer. |
+| [Session](#!ingesters/session.md) | Ingest large records into a single entry. |
+| [Simple Relay](#!ingesters/simple_relay.md) | Ingest any text over TCP/UDP, syslog, and more. |
+| [Windows Events](#!ingesters/winevent.md) | Collect Windows events. |
+| [collectd](#!ingesters/collectd.md) | Ingest collectd samples. |
```

Comment:
Let's just put this below "Amazon SQS"... computer-alphabetic sort tends to confuse people